packages/datadog-plugin-memcached/src/index.js

```diff
@@ -9,15 +9,20 @@ class MemcachedPlugin extends CachePlugin {
   start ({ client, server, query }) {
     const address = getAddress(client, server, query)
 
+    const meta = {
+      'out.host': address[0],
+      [CLIENT_PORT_KEY]: address[1]
+    }
+
+    if (this.config.memcachedCommandEnabled) {
```

Comment:
Since the option is for memcached, specifying "memcached" in the name is redundant.